src/components/CommandBar/CommandBar.tsx

```diff
@@ -146,7 +146,7 @@ export const CommandBar = () => {
           leaveTo="opacity-0 scale-95"
         >
           <WrapperComponent.Panel
-            className="relative z-50 pointer-events-auto w-full max-w-xl pt-2 mx-auto border rounded rounded-tl-none shadow-lg bg-chalkboard-10 dark:bg-chalkboard-100 dark:border-chalkboard-70"
+            className="relative z-50 pointer-events-auto w-full max-w-2xl pt-2 mx-auto border rounded rounded-tl-none shadow-lg bg-chalkboard-10 dark:bg-chalkboard-100 dark:border-chalkboard-70"
```

Comment:
@franknoirot I'm not sold on this change it feels weird, maybe just me? 

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/b1a69321-5827-4655-91ac-032e0f83af35" />

I'm wondering if we shouldn't move the progress button to a footer instead? Potentially taking the chance to make the step back button more visible by moving it there as well?

Reply:
Maybe something like this? 

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/c0576103-6c83-4c2c-85c7-bb4eae355c3f" />

Reply:
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/23d62e70-02db-4c4f-a200-e44f7c9d7318" />

Reply:
Merged the branch in for the footer arg. Played with layout a lil more

<img width="696" alt="image" src="https://github.com/user-attachments/assets/0d619747-d33b-464e-9f65-4b10a17a37d0" />


https://github.com/user-attachments/assets/77b8faf1-a36c-4045-abb9-dcdffd1bf857

Reply:
Hahaha sorry I didn't realize my screenshot was pretty stale!!

Reply:
I've been thinking about Modes all of yesterday pretty much, trying to see if we should go full on optional args or keep the command bar opionated about the couple prime paths we want to have the user follow